core/src/toga/app.py

```diff
@@ -347,6 +347,10 @@ def __init__(
     def _create_impl(self):
         return self.factory.App(interface=self)
 
+    @property
+    def screens(self):
+        return self._impl.get_screens()
```

Comment:
This means `get_screens()` is returning interface instances. It should be returning impl instances. I raised a similar issue with `window.get_screen()`

Reply:
I have addressed this issue in the latest commit.